terraso_backend/apps/soil_id/models/depth_dependent_soil_data.py

```diff
@@ -23,3 +23,16 @@ class DepthDependentSoilData(BaseModel):
     soil_data_input = models.ForeignKey(SoilData, on_delete=models.CASCADE)
     depth_top = models.IntegerField(blank=True)
     depth_bottom = models.IntegerField(blank=True)
+    slope = models.IntegerField(blank=True)
+    texture = models.CharField(blank=True)
```

Comment:
for texture:
- for all the enum char fields, we should use the `choices` argument to make sure the database only accepts valid values. see e.g. the privacy field in the project model: https://github.com/techmatters/terraso-backend/blob/7d5820d221c448a72916e05de20d77402a32d2ac/terraso_backend/apps/project_management/models/projects.py#L43-L54
- there is a second enum for rock fragment volume under the soil texture spec